types/connect-timeout/index.d.ts

```diff
@@ -37,6 +37,6 @@ declare module "connect-timeout" {
         }
     }
 
-    function e(timeout: string, options?: e.TimeoutOptions): express.RequestHandler;
+    function e(timeout: number | string, options?: e.TimeoutOptions): express.RequestHandler;
```

Comment:
Might be worth adding a comment to this function to explain what a number (and a string for that matter) represents, e.g. copying from the README:

```suggestion
    /**
     * @summary Returns middleware that times out in time milliseconds. time can also be a string accepted by the ms module. On timeout, req will emit "timeout".
     */
    function e(time: number | string, options?: e.TimeoutOptions): express.RequestHandler;
```